src/undo.h

```diff
@@ -310,11 +317,20 @@ struct CSidechainUndoData
             res += strprintf("prevInclusionHeight=%d\n", prevInclusionHeight);
         }
 
-        res += strprintf("scFees.size()=%u\n", scFees.size());
-        for(const auto& entry: scFees)
-        {
-           res += strprintf("scFtFee=%d.%08d - ", entry.forwardTxScFee / COIN, entry.forwardTxScFee % COIN);
-           res += strprintf("scMbtrFee=%d.%08d\n", entry.mbtrTxScFee / COIN, entry.mbtrTxScFee % COIN);
+        if (contentBitMask & AvailableSections::NONCEASING_CERT_DATA) {
+            res += strprintf("scFees.size()=%u\n", scFees_v2.size());
```

Comment:
I see in the V2-case the res variable would contain submissionHeight (conversely from non V2-case), but why not emphasizing it with `strprintf("scFees.size()=%u\n", scFees_v2.size())`

Reply:
Keeping the same name for the new structure version should not break the compatibility. For v2 SC, the basic fee data will be listed under the same structure as v1, with just an additional field.